code/game/objects/items/plushes.dm

```diff
@@ -676,3 +677,18 @@
 	item_state = "moffplush"
 	attack_verb = list("fluttered", "flapped")
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
+///Used to track how many people killed themselves with item/toy/plush/moth
+	var/suicide_count = 0
+
+/obj/item/toy/plush/moth/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] stares deeply into the eyes of [src] and it begins consuming [user.p_them()]!  It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	suicide_count++
+	if(suicide_count < 3)
+		desc = "A plushie depicting an unsettling mothperson. After killing [suicide_count] [suicide_count == 1 ? "person" : "people"] it's not looking so huggable now..."
+	else
+		desc = "A plushie depicting a creepy mothperson. It's killed [suicide_count] people! I don't think I want to hug it any more!"
+		divine = TRUE
+		resistance_flags = INDESTRUCTIBLE | FIRE_PROOF | ACID_PROOF | LAVA_PROOF
+	playsound(src, 'sound/voice/moth/scream_moth.ogg', 100)
+	user.dust(just_ash = FALSE, drop_items = TRUE)
```

Comment:
you could probably forcemove the plushie to an adjacent tile to avoid the sound spam actually

Reply:
Seems really weird on first thought. In the current state it just accepts that there will be screaming, and changed the sound so that it, at a minimum, screams once. 
I'm worried that forcefully moving the plushie to an adjacent tile might lead to other confusing situations, but I can't think of any off the top of my head at 1 AM. 
Thanks for the suggestion. I'll definitely continue thinking about it